catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/integration/test/CatalogHiveIT.java

```diff
@@ -2,7 +2,7 @@
  * Copyright 2023 Datastrato Pvt Ltd.
  * This software is licensed under the Apache License version 2.
  */
-package com.datastrato.gravitino.integration.test.catalog.hive;
+package com.datastrato.gravitino.catalog.integration.test;
```

Comment:
Why do you remove "hive" in package path?

Reply:
I moved it to test some cases and forgot to revert it.

Reply:
done